### DISCUSSION TITLE
User does not see public files

### DISCUSSION THREAD

**Discussion by laser-vitali** [SergiX44/XBackBone#388] [General]
Hi,
I installed the latest built 3.5.0, created two users and one admin accounts.
Admin and users uploaded pdf files, one of each, three in total.
So, now admin can see all three files and manage them, for example, delete them. It's good.
But users can see only one file they uploaded and can't see files from others, even all uploaded files are public...
Do I miss something? Please help. Thanks.

**Comment by sergix44** [comment]
For public it means that are accessible by guests, but they still need a link to the media.

**Reply by laser-vitali** [reply]
Hi, thank you for getting back to me.
That's not what I thought. :(
Is it possible to make a folder that is seen by all registrated users and where they can upload their files, but only an administrator can delete them?
Thanks.
